openquake/qa_tests_data/event_based_risk/case_1/agg_curve-rlz-000_6879.csv

```diff
@@ -0,0 +1,41 @@
+loss_type,loss,poe
```

Comment:
Should the final number of the filename (6879) be removed? Please check about it here and for the other output files.